src/database/schema.ts

```diff
@@ -61,3 +75,74 @@ export const sessionRelations = relations(session, ({ one }) => ({
     references: [users.uuid],
   }),
 }))
+
+export const jobs = pgTable('jobs', {
+  ...createdAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  user_uuid: uuid('user_uuid')
+    .notNull()
+    .references(() => users.uuid),
+  name: varchar('name'),
+  status: varchar('status').notNull(),
+  data: jsonb('data'),
+  errors: jsonb('errors'),
+})
+
+export const jobsRelations = relations(jobs, ({ one }) => ({
+  user: one(users, {
+    fields: [jobs.user_uuid],
+    references: [users.uuid],
+  }),
+}))
+
+export const tickets = pgTable('tickets', {
+  ...createdAt,
+  ...updatedAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  title: varchar('title'),
+  description: varchar('description'),
+  user_uuid: uuid('user_uuid').references(() => users.uuid),
+  creator_uuid: uuid('creator_uuid')
+    .notNull()
+    .references(() => users.uuid),
```

Comment:
why those two?

I guess user_uuid is like `asignee` ? I'd call it that way then - since user_uuid is more like creator to me

Reply:
I see it's called like that in relations but also here should be worthy